lib/galaxy/jobs/output_checker.py

```diff
@@ -15,6 +15,9 @@
 )
 
 
+ERROR_PEAK = 2000
```

Comment:
could the 8x larger peak be a problem for the log size?

Reply:
We can make it configurable. But 2000 is still less than your average traceback.